test/extended/builds/dockerfile.go

```diff
@@ -119,6 +122,19 @@ USER 1001
 				}
 				o.Expect(err).NotTo(o.HaveOccurred())
 			})
+
+			g.It("testing build image with invalid dockerfile content", func() {
+				// https://bugzilla.redhat.com/show_bug.cgi?id=1694867
+				g.By("creating a build")
+				err := oc.Run("new-build").Args("--binary", "--strategy=docker", "--name=centos").Execute()
+				o.Expect(err).NotTo(o.HaveOccurred())
+				br, err := exutil.StartBuildAndWait(oc, "centos", fmt.Sprintf("--from-dir=%s", dockerfileAdd))
+				br.AssertFailure()
+				g.By("build log should have error about no file or directory")
+				logs, _ := br.Logs()
```

Comment:
iirc `br.Logs()` only returns an error if we can't get the build logs. Add another check to ensure an error doesn't occur here.

Reply:
Yes, you are right, added error check,thanks for your advice.